docs/architecture/adr-0014-api-documentaton-revisited.md

```diff
@@ -0,0 +1,104 @@
+* Angular CLI seems pretty solid.  I haven't run into any of the usual webpack issues using it.
+  There is now an RC.0, which means that no project code changes should be required in order to 
+  upgrade to v1
+* Typescript is amazing
+* Supports Redux
+* Html templates with javascript controllers
+* Maintained by Google: The direction that a central team provides can be valuable
+
+##### Cons
+* We need to learn the framework while we develop
+  * Ecosystem is still a bit young compared to Angular 1.X and Ember
+  * Need to learn typescript
+  * Many core concepts have changed from Angular 1.X (services, scope, etc). 
+    The framework is more like a plug and play setup for react with angular influence
+* Maintained by Google: Its success leans heavily on Google's continued support (see )
+* Not auto-generated
+
+
+#### Ember
+
+##### Pros
+* Existing prototype
+* Stable, open source framework that isn't going to disappear unless something
+  much better comes along and invalidates the work that's been done
+* Performance aligned with other mainstream frameworks
+* Supports typescript
+* awesome CLI
+* Supports Redux
+* Good ecosystem - well tested modules
+* Good upgrade tooling
+* html with inline javascript
+
+##### Cons
+* Learning ember - one more framework to keep track of
+* A classic open source project that is maintained by the community. 
+* Not auto generated
+
+## Proposed Solution
+We will write the api documentation as a traditional SPA using Angular 2, using either
+Webpack or Angular CLI as a build tool. This should allow us to accomplish our flexibility
+goals while also allowing us to test new technology in a place where it will have less impact.
+
+## Consequences / Tradeoffs
+### Libraries
+#### Pros
+Angular 2 solves a lot of issues that Angular 1.X has, and allows for cleaner, more maintainable code.
+Angular 2's design was highly influenced by the progress made in frameworks such as React and Ember,
+which addresses many of the issues that people had with Angular 1.X such as poor performance and
+highly complex controller and template logic
+#### Cons
+Angular 2 is a younger framework, so some libraries that we rely on in Angular 1.X may not exist, 
+or may exist in a less stable form.
+
+Examples that come to mind are component frameworks - many of them exist for Angular 1.X, 
+but the selection seems a bit fragmented or less fully featured for Angular 2. 
+With that said, writing custom components isn't the end of the world when your application 
+is structured around creating composable components.  We rarely need all of the features 
+which component libraries offer, and frequently have to hack our way around their limitations.
+
+### Experience
+#### Pros
+One of our team goals is to test new frameworks on smaller projects before we commit to converting our larger ones. 
+We've established that for Hunchlab and Raster Foundry, we will eventually want to move from Angular 1.X to Angular 2.X
```

Comment:
If this is the main driver of the decision then I think a lot of the above description of alternative frameworks can be removed. That is, if we've already determined that both of our major projects will use Angular 2 someday then our overriding concern should be gaining experience with that new framework and I'm not sure we need to include detailed evaluations of Angular 1.x, React, or Ember at all. If that's true then this should be moved into the "Context" section and the sections on React and Ember and Angular 1.x can be omitted (or shortened significantly).